processing/src/main/java/io/druid/segment/MetricHolder.java

```diff
@@ -155,24 +75,14 @@ public static MetricHolder fromByteBuffer(ByteBuffer buf, ObjectStrategy strateg
           holder.complexType = GenericIndexed.read(buf, serdeForType.getObjectStrategy());
         }
         break;
+      case LONG:
+      case DOUBLE:
```

Comment:
Why this change?

Reply:
Apparently MetricHolder is used only when dealing with legacy segment format, in which long- and double-typed columns doesn't exist